server/schedulers/utils.go

```diff
@@ -73,6 +74,9 @@ func shouldBalance(cluster opt.Cluster, source, target *core.StoreInfo, region *
 	targetInfluence := opInfluence.GetStoreInfluence(targetID).ResourceProperty(kind)
 	sourceScore := source.ResourceScore(kind, cluster.GetHighSpaceRatio(), cluster.GetLowSpaceRatio(), sourceInfluence-tolerantResource)
 	targetScore := target.ResourceScore(kind, cluster.GetHighSpaceRatio(), cluster.GetLowSpaceRatio(), targetInfluence+tolerantResource)
+	opInfluenceStatus.WithLabelValues(scheduleName, strconv.FormatUint(sourceID, 10), "source").Set(float64(sourceInfluence))
+	opInfluenceStatus.WithLabelValues(scheduleName, strconv.FormatUint(targetID, 10), "target").Set(float64(targetInfluence))
+	tolerantResourceStatus.WithLabelValues(scheduleName, strconv.FormatUint(sourceID, 10), strconv.FormatUint(targetID, 10)).Set(float64(tolerantResource))
```

Comment:
There are n*n labels in these metrics,  is this very meaningful?

Reply:
Usually it is hidden, just open the relevant label in debug.

Reply:
I think better to remove it too. Looks not very energy saving.

Reply:
emmm but we can not get more detail when occur many balance operator without it

Reply:
can we output the debug log below no matter `shouldBalance` is true or false?

Reply:
Could it help debug? (the log level can be set online)

Reply:
it is difficult to draw a line from log to observe. how about to set the balance region metrics switch online?

Reply:
Switch metrics online seems too complicated :-( /cc @nolouch  Any idea?

Reply:
I think it is not complicated. Only add a bool option to switch whether record tolerant size and op influence?

Reply:
I think to add an option is ok, because, in the process of our development, we really need to add some metrics to test the effect of balance, such as the traffic of some regions. but we may need to hide this option to the user. cc @disksing

Reply:
I have added option. PTAL @nolouch @disksing